PackCommandLine.bat

```diff
@@ -0,0 +1,17 @@
+@Set SOLUTION=XamlStyler.sln
+
```

Comment:
I am getting error VSSDK1000 when trying to run this script. It appears to be related to a conflict between using a x64 version of msbuild.exe and a x86 version of VSCT.exe (the latter only shipped x86). A solution is just running it via x86 command line, but I believe we can force x86 msbuild.exe in an x64 environment to make this a little more robust.